0018-connector-risk-mitigations/0018-connector-risk-mitigations.md

```diff
@@ -17,25 +17,28 @@ The main risk connectors face in Interledger is being unable to fulfill the inco
 Possible mitigations include:
 
 * **Connecting Reliable Ledgers** - Connectors should choose the ledgers they connect in part based on the reliability and speed of fulfillment notifications and the time the ledgers require to process fulfillments once they are submitted
-* **Adjusting Transfer Expiry Window** - Connectors determine the window of time they require between when the incoming and outgoing transfers expire. Connectors should set this window such that they are confident that they will be able to deliver the fulfillment in time even if the outgoing transfer is executed at the last possible moment.
 * **Denial of Service Protections** - Connectors should take standard [Denial of Service](https://en.bitcoin.it/wiki/Weaknesses#Denial_of_Service_.28DoS.29_attacks) attack prevention steps to prevent attackers from overloading the connectors' servers with network traffic
```

Comment:
It may be worth mentioning that this is both IP-level DoS protections and within-ILP protections